cmd/apps/certmanager_app.go

```diff
@@ -120,7 +121,7 @@ func MakeInstallCertManager() *cobra.Command {
 				return err
 			}
 		} else {
-			err = templateChart(chartPath, "cert-manager", namespace, outputPath, "values.yaml", "v0.12.0", nil)
+			err = templateChart(chartPath, "cert-manager", namespace, outputPath, "values.yaml", nil)
```

Comment:
I think this reverted the version or pinning

Reply:
its moved into fetchChart (for helm2)